### PR TITLE
APIS-7086 - Part 1 - Session proxy

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -7,15 +7,20 @@ object AppDependencies {
 
   private val bootstrapVersion = "9.1.0"
   private val hmrcMongoVersion = "2.2.0"
+  private val tpdDomainVersion = "0.6.0-SNAPSHOT"
```
Comment (Tobias1087) [inline project/AppDependencies.scala:10]: does this want to be a concrete version now?

**Review by Tobias1087 — APPROVED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (AndySpaven) [reply project/AppDependencies.scala:10]: Probably but there are so many moving parts what's another problem :-)

**Review by Tobias1087 — APPROVED** [review]

**Review by Tobias1087 — DISMISSED** [review]

**Comment by AndySpaven** [comment]
test me

**Comment by AndySpaven** [comment]
retest

**Comment by AndySpaven** [comment]
test me please

**Comment by AndySpaven** [comment]
test me please

**Pull request merged**